build/Targets/Settings.targets

```diff
@@ -132,13 +132,13 @@
   <Choose>
     <When Condition="'$(VisualStudioVersion)' == '15.0'">
       <PropertyGroup>
-        <VisualStudioBuildToolsNuGetPackagePath>$(NuGetPackageRoot)\Microsoft.VSSDK.BuildTools\15.0.25201-dev15preview2</VisualStudioBuildToolsNuGetPackagePath>
+        <VisualStudioBuildToolsNuGetPackagePath>$(NuGetPackageRoot)\Microsoft.VSSDK.BuildTools\15.0.25604-Preview4</VisualStudioBuildToolsNuGetPackagePath>
```

Comment:
Right now packages marked as "fixed" in RepoData.json can't be entries in Dependencies.props.  Hence you have to hard code a version here vs being able to say `$(MicrosoftVSSDKBuildToolsVersion)`